README.md

```diff
@@ -19,307 +28,41 @@ For more information, see
 | Sponsor development on OpenCollective           | [![Financial Contributors](https://opencollective.com/opensourcebim/tiers/badge.svg)](https://opencollective.com/opensourcebim/)             |
 | Docker hub                                      | [![Docker Pulls](https://img.shields.io/docker/pulls/aecgeeks/ifcopenshell)](https://hub.docker.com/r/aecgeeks/ifcopenshell)                 |
 
-Prerequisites
```

Comment:
For the removed sections can we have direct links to some of the doc content for:

- All building/installing on X
- Quick start